ansible_runner/runner_config.py

```diff
@@ -197,10 +197,6 @@ def prepare_env(self):
             # Still need to pass default environment to pexpect
             self.env = os.environ.copy()
 
-        # extravars dict passed in via the interface API takes precedence over on-disk
-        if not self.extra_vars and self.loader.isfile('env/extravars'):
-            self.extra_vars = self.loader.abspath('env/extravars')
```

Comment:
Should still keep the self.extra_vars so if somewhere else in the runner code someone wants to reference it.

Reply:
in this case we're just *not* using this to populate the file if it's not given as an argument.